awx/main/tests/functional/models/test_job_options.py

```diff
@@ -12,24 +11,11 @@ def test_clean_credential_with_ssh_type(credentialtype_ssh, job_template):
     )
     credential.save()
 
-    job_template.credential = credential
+    job_template.credentials.add(credential)
     job_template.full_clean()
 
 
 @pytest.mark.django_db
-def test_clean_credential_with_invalid_type_xfail(credentialtype_aws, job_template):
```

Comment:
No longer necessary now that we've consolidated everything under `.credentials`.